docs/processes/release/tasks/versions.md

```diff
@@ -53,7 +53,7 @@ need to be correspondingly updated. See below for more files to be updated and s
 [icu4c/source/data/misc/icuver.txt](https://github.com/unicode-org/icu/blob/main/icu4c/source/data/misc/icuver.txt)
 needs to be updated with the correct version number for ICU and its data.
 
-#### Since ICU 68
+#### Since ICU 68, but before ICU 77
```

Comment:
Should there be a "Since ICU 77" version added?  Even if it says to look somewhere else?

The README in the codebase in the `cldr-to-icu` folder points to the `cldr-icu.md` page in the ICU User Guide...

Reply:
"Since ICU 77" would be a NOP :-)

Reply:
I think it's worth adding, even if were a no-op. But the fact that the configuration file is no longer in `build-icu-data.xml` but instead in some new thing called `config.xml` is worth pointing out. And you can also point to the README which has usage tips and examples. It's simple & not much, because it doesn't need to be much, and it's more significant than nothing.

Reply:
DONE